tests/system/Makefile

```diff
@@ -0,0 +1,27 @@
+PREFIX ?= riscv32-unknown-elf-
+ARCH = -march=rv32imaf
```

Comment:
Don't specify "f" extension. Minimize when possible.

Reply:
I would refine it as `-march=rv32izicsr`